File: Auth_Services/Auth_Services/DTOs/TurmaDTO.cs
namespace Auth_Services.DTOs
{
    public class TurmaDTO
    {
        public int TurmaId { get; set; }
        public string TurmaName { get; set; }
        public int CourseId { get; set; }
        public string CourseName { get; set; }
        public int isDeleted { get; set; }
        public DateTime? DateStart { get; set; } // Added
        public DateTime? DateEnd { get; set; }   // Added
    }
}

File: AndroidApp/app/src/main/java/com/example/nts_app/network/TurmaDTO.kt
package com.example.nts_app.network

data class TurmaDTO(
    val turmaId: Int,
    val turmaName: String,
    val courseId: Int,
    val courseName: String,
    val isDeleted: Int,
    val dateStart: String?,
    val dateEnd: String?
)